docs/src/content/docs/reference/Config File/Steps/release.md

```diff
@@ -21,15 +21,15 @@ If the config file is using the `[packages.{name}]` syntax,
 each package gets its own tag in the format `{name}/v{version}` (this is the syntax required for Go modules).
 See examples below for more illustration.
 
-## GitHub release notes
+## Release notes
 
 There are several different possible release notes formats:
 
 1. If run after a [`PrepareRelease`] step in the same workflow, the release notes will be the same as the changelog section created by [`PrepareRelease`] even if there is no changelog file configured—with the exception that headers are one level higher (for example, `####` becomes `###`).
 2. If run in a workflow with no [`PrepareRelease`] step before it (the new version was set another way), and there is a changelog file for the package, the release notes will be taken from the relevant changelog section. This section header must match exactly what [`PrepareRelease`] would have created. Headers will one level higher (for example, `####` becomes `###`).
 3. If run in a workflow with no [`PrepareRelease`] step before it (the new version was set another way), and there is no changelog file for the package, the step will use GitHub's automatic release notes generation.
```

Comment:
Is there an equivalent of auto release notes for Gitea? We should clarify this sentence to either be generic or clarify that this only happens for GitHub/

Reply:
The features are identical, besides what is mentioned in the notes up top. I likely just missed this ^^